crates/pkgcraft-tools/src/subcmds/pkg/source.rs

```diff
@@ -430,7 +432,7 @@ fn cumulative(limit: u32, targets: PkgTargets, cmd: &Command) -> anyhow::Result<
         writeln!(stdout, "{value}")?;
     }
 
-    Ok(ExitCode::from(failed as u8))
+    Ok(ExitCode::from(0))
```

Comment:
This discards the ability for invalid packages to influence the returned error status, I'm not sure that's a good thing since it tries to match most of the other tools which work like that.

Reply:
I forgot to re-enabled it.